src/renderer/widgets/DelegateDetails/lib/utils.ts

```diff
@@ -26,3 +30,28 @@ export const getIdentityList = (identity: Identity) => {
     }
   }, []);
 };
+
+export const getDelegationsList = (delegations: Delegation[]) => {
+  const map = new Map();
+
+  for (const delegation of delegations) {
+    const currentMap = map.get(delegation.delegator);
```

Comment:
`currentMap` name looks incorrect here, because it's not map, just object. Maybe call it like `currentItem` or `delegationItem` or smth else